tests/Unit/ItemTest.php

```diff
@@ -2834,7 +2834,7 @@ public function test_get_longitude(string $data, float $expected): void
     }
 
     /**
-     * @return array<array{string, string}>
+     * @return array<string, array<int, string|null>>
```

Comment:
We should keep the [tuple syntax](https://phpstan.org/writing-php-code/phpdoc-types#array-shapes):

```suggestion
     * @return array<string, array{string, string|null}>
```

Reply:
Done ✅